src/SchemaGenerator.cpp

```diff
@@ -146,6 +146,12 @@ static_assert(graphql::internal::MinorVersion == )cpp"
 		}
 	}
 
+		headerFile << R"cpp(
+
+using namespace std::literals;
```

Comment:
I'd prefer not to add a `using namespace` statement in the header, although this is at least scoped to the generated namespace. Consider leaving this out and wrapping the string literals in explicit `std::string_view` constructors, like in `static_typename` instead of appending the `sv` literal operator.

It looks like this is copied from the `sourceFile` generation. Could the `sourceFile` use this instead of declaring its own `std::array`? Or could the entire mapping function be made `constexpr` and moved to the header? Then you could also keep the `using namespace std::literals;` but in the scope of the `constexpr` function.